services-js/access-boston/src/client/TextInput.tsx

```diff
@@ -19,18 +37,33 @@ interface Props extends InputHTMLAttributes<HTMLInputElement> {
  *
  * All props besides "label" and "error" are passed along as props for the
  * <input> element.
+ *
+ * TODO(finh): Merge this with the react-fleet TextInput.
```

Comment:
When we get to this point let's talk through theory and come up with consensus, since I went with a different approach with the current react-fleet version.

Reply:
👍